twenty-first/src/math/polynomial.rs

```diff
@@ -134,28 +146,36 @@ where
     /// Extracted from `cargo bench --bench poly_mul` on mjolnir.
     const FAST_MULTIPLY_CUTOFF_THRESHOLD: isize = 1 << 8;
 
-    /// [Divide and conquer batch evaluation](Self::divide_and_conquer_batch_evaluate)
-    /// is slower than [iterative evaluation](Self::iterative_batch_evaluate)
-    /// on domains smaller than this threshold
-    const DIVIDE_AND_CONQUER_EVALUATE_CUTOFF_THRESHOLD: usize = 1 << 5;
-
     /// Computing the [fast zerofier][fast] is slower than computing the [smart zerofier][smart] for
     /// domain sizes smaller than this threshold. The [naïve zerofier][naive] is always slower to
     /// compute than the [smart zerofier][smart] for domain sizes smaller than the threshold.
     ///
-    /// Extracted from `cargo bench --bench zerofier` on mjolnir.
+    /// Extracted from `cargo bench --bench zerofier`.
     ///
     /// [naive]: Self::naive_zerofier
     /// [smart]: Self::smart_zerofier
     /// [fast]: Self::fast_zerofier
-    const FAST_ZEROFIER_CUTOFF_THRESHOLD: usize = 200;
+    const FAST_ZEROFIER_CUTOFF_THRESHOLD: usize = 100;
 
     /// [Fast interpolation](Self::fast_interpolate) is slower than
     /// [Lagrange interpolation](Self::lagrange_interpolate) below this threshold.
     ///
     /// Extracted from `cargo bench --bench interpolation` on mjolnir.
     const FAST_INTERPOLATE_CUTOFF_THRESHOLD: usize = 1 << 8;
 
+    /// Regulates the recursion depth at which [Fast modular coset interpolation](Self::fast_modular_coset_interpolate)
+    /// is slower and switches to [Lagrange interpolation](Self::lagrange_interpolate).
+    const FAST_MODULAR_COSET_INTERPOLATE_CUTOFF_THRESHOLD_PREFER_LAGRANGE: usize = 1 << 8;
+
+    /// Regulates the recursion depth at which [Fast modular coset interpolation](Self::fast_modular_coset_interpolate)
+    /// is slower and switches to [INTT](ntt::intt)-then-[reduce](Self::reduce).
```

Comment:
Violates the 100 character line width and the 80 character comment-length width.

```suggestion
    /// Regulates the recursion depth at which
    /// [Fast modular coset interpolation](Self::fast_modular_coset_interpolate) is
    /// slower and switches to [INTT](ntt::intt)-then-[reduce](Self::reduce).
```

Reply:
42999c2f